itl/command-icinga.conf

```diff
@@ -39,3 +39,14 @@ object CheckCommand "exception" {
 object CheckCommand "sleep" {
     import "sleep-check-command"
 }
+
+object CheckCommand "ifw-api" {
+	import "ifw-api-check-command"
+
+	vars.ifw_api_command = "$command.name$"
+	vars.ifw_api_arguments = "$command.arguments$"
+	vars.ifw_api_ignore_arguments = [ "-C", "-ExecutionPolicy", "-NoLogo", "-NoProfile" ]
+	vars.ifw_api_host = "localhost"
+	vars.ifw_api_port = 5668
+	vars.ifw_api_sni = NodeName
```

Comment:
This way, this will be somewhat annoying to configure because you can't just set the host variable, you'd pretty much always have to set both. In any case except the default, using the host for SNI and certificate validation by default would make sense.

Also not sure if I'm happy with the name, as it affects more than just sending the SNI extension in the handshake.

Reply:
1. People (and the IfW PS wizard) already set endpoint name AND IP address in the Icinga 2 zones config for the case DNS breaks
2. Sure, it also asserts the hostname, as every product handles the SNI